docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -189,12 +185,21 @@ The ticket request requires the token in one of the following formats:
 - Cookie named apimlAuthenticationToken.
 - Bearer authentication
 
-The request takes one parameter named `applicationName`, the name of the application for which the PassTicket should be generated. This parameter must be supplied.
+The request takes the `applicationName` parameter, which is the name of the application for which the PassTicket should be generated. This parameter must be supplied.
```

Comment:
The mandatory "applicationName" parameter is expected in the JSON payload of the request.